R/latent_individual.R

```diff
@@ -104,46 +104,35 @@ is_latent_individual <- function(data) {
   inherits(data, "epidist_latent_individual")
 }
 
-#' Check if data has the `epidist_latent_individual` class
-#'
-#' @param data A `data.frame` containing line list data
-#' @param family Output of a call to `brms::brmsfamily()`
-#' @param ... ...
+#' Create the model-specific component of an `epidist` custom family
 #'
-#' @method epidist_family epidist_latent_individual
+#' @inheritParams epidist_family_model
+#' @param ... Additional arguments passed to method.
+#' @method epidist_family_model epidist_latent_individual
 #' @family latent_individual
 #' @export
-epidist_family.epidist_latent_individual <- function(data,
-                                                     family = "lognormal",
-                                                     ...) {
+epidist_family_model.epidist_latent_individual <- function(
+  data, family, ...
+) {
   epidist_validate(data)
```

Comment:
Is there a reason not to move this into the epidist_family wrapper?

Reply:
No good reason, so will move

Reply:
[7530cdb](https://github.com/epinowcast/epidist/pull/365/commits/7530cdb8cb5e5a1ac5762141224fd8ae816b55e8)